tests/numpy/test_stack.py

```diff
@@ -91,6 +91,69 @@ def test_atleast_3d(nps_app_inst):
 
     try_multiple_nd(np.atleast_3d, nps.atleast_3d)
 
+
+def test_hstack(nps_app_inst):
```

Comment:
These tests look like they're doing the same thing. Can we consolidate all of these into a single test?